roles/nova-common/defaults/main.yml

```diff
@@ -50,7 +50,7 @@ nova:
   # librbd1 version should match the ceph client version
   librbd1_version: "{{ ceph.client_version[ursula_os] }}"
   qemu_system_package: qemu-system-x86
-  glance_endpoint: http://{{ endpoints.main }}:9393
+  glance_endpoint: https://{{ endpoints.main }}:9393
```

Comment:
we cannot change this to https. This is backend glance endpoint which is http. All our services backend ports are http.